portfolio/src/main/webapp/style.css

```diff
@@ -25,6 +25,64 @@ body {
   max-width: 100%;
 }
 
+#intro,
+.intro-wrapper {
+  align-items: center;
+  display: flex;
+  justify-content: center;
+}
+
+.intro-wrapper {
+  height: 50vh;
+  text-align:center;
```

Comment:
add a space after `:`, here and elsewhere where it is missing

Reply:
Rectified here and elsewhere in commit ab13304